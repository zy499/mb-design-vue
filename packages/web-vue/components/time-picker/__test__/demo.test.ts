import demoTest from '../../../scripts/demo-test';

demoTest('time-picker');
