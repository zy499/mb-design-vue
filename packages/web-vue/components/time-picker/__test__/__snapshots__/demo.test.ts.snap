// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<time-picker> demo: render [basic] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [control] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [default-value] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px; margin-right: 24px; margin-bottom: 24px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-range mb-picker-size-medium\\" style=\\"width: 252px; margin-bottom: 24px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input mb-picker-input-active\\"><input placeholder=\\"开始时间\\"></div><span class=\\"mb-picker-separator\\"> - </span>
  <div class=\\"mb-picker-input\\"><input placeholder=\\"结束时间\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [disable-confirm] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-range mb-picker-size-medium\\" style=\\"width: 252px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input mb-picker-input-active\\"><input placeholder=\\"开始时间\\"></div><span class=\\"mb-picker-separator\\"> - </span>
  <div class=\\"mb-picker-input\\"><input placeholder=\\"结束时间\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [disabled] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium mb-picker-disabled\\" style=\\"margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input disabled=\\"\\" placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-range mb-picker-size-medium mb-picker-disabled\\" style=\\"width: 252px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input mb-picker-input-active\\"><input disabled=\\"\\" placeholder=\\"开始时间\\"></div><span class=\\"mb-picker-separator\\"> - </span>
  <div class=\\"mb-picker-input\\"><input disabled=\\"\\" placeholder=\\"结束时间\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [disabled-time] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-range mb-picker-size-medium\\" style=\\"width: 252px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input mb-picker-input-active\\"><input placeholder=\\"开始时间\\"></div><span class=\\"mb-picker-separator\\"> - </span>
  <div class=\\"mb-picker-input\\"><input placeholder=\\"结束时间\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [extra] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [format] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 130px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [rangepicker] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-range mb-picker-size-medium\\" style=\\"width: 252px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input mb-picker-input-active\\"><input placeholder=\\"开始时间\\"></div><span class=\\"mb-picker-separator\\"> - </span>
  <div class=\\"mb-picker-input\\"><input placeholder=\\"结束时间\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [size] correctly 1`] = `
"<div style=\\"margin-bottom: 20px;\\"><span class=\\"mb-radio-group-button mb-radio-group-size-undefined mb-radio-group-direction-horizontal\\"><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"mini\\"><span class=\\"mb-radio-button-content\\">mini</span></label><label class=\\"mb-radio-button mb-radio-checked\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"small\\"><span class=\\"mb-radio-button-content\\">small</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"medium\\"><span class=\\"mb-radio-button-content\\">medium</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"large\\"><span class=\\"mb-radio-button-content\\">large</span></label></span></div>
<div class=\\"mb-picker mb-picker-size-small\\" style=\\"width: 194px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\">
    <!--v-if--><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span>
  </div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [step] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->"
`;

exports[`<time-picker> demo: render [use-12-hours] correctly 1`] = `
"<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-size-medium\\" style=\\"width: 194px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input\\"><input placeholder=\\"请选择时间\\" class=\\"mb-picker-start-time\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->
<div class=\\"mb-picker mb-picker-range mb-picker-size-medium\\" style=\\"width: 300px; margin: 0px 24px 24px 0px;\\" visible=\\"false\\" editable=\\"true\\">
  <div class=\\"mb-picker-input mb-picker-input-active\\"><input placeholder=\\"开始时间\\"></div><span class=\\"mb-picker-separator\\"> - </span>
  <div class=\\"mb-picker-input\\"><input placeholder=\\"结束时间\\"></div>
  <div class=\\"mb-picker-suffix\\"><span class=\\"mb-icon-hover mb-picker-icon-hover mb-picker-clear-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-picker-suffix-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-clock-circle\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M24 14v10h9.5m8.5 0c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6s18 8.059 18 18Z\\"></path></svg></span></div>
</div>
<!---->"
`;
